### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by kr-sarvesh**
Home-page of eCommerce added
like to hear reviews

[red-estore.netlify.app](https://red-estore.netlify.app/)

### CODE REVIEW

File: homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+}
+.header .row {
+  margin-top: 70px;
+}
+.categories {
+  margin: 60px 0;
+}
+.col-3 {
+  flex-basis: 30%;
+  min-width: 250px;
```
Comment (Koushith) [inline homepage/homepage.css]: try using relative units here so that it will scale when a user visits the site on larger screens.

**Review by Koushith — APPROVED** [review]
Loved the UI. good to go. LGTM.

**Review by aurobindoGupta — APPROVED** [review]
nice

**Review by krgarima — COMMENTED** [review]
Check the color contrast ratio of nav anchor tags text. It is around 2. It should be between 4 to 8 for accessibility. 
Same goes for logo. Choose a darker shade of red for the text 'RED' in logo to increase readibility. Rest is fine.

File: homepage/homepage.css (hunk 1 above)
Comment (kr-sarvesh) [reply homepage/homepage.css]: thank you will implement

**Pull request merged**